src/hyper.rs

```diff
@@ -0,0 +1,15 @@
+use bytes::{Bytes, BytesMut};
+use futures::TryStreamExt;
+use hyper13::{Body, Error};
+
+// Can be eliminated once we start using `bytes` crate with the same version as `hyper`.
```

Comment:
Mind prefixing this with `TODO: `? @Hoverbear @lukesteensen how are we handling these types of tasks?

Reply:
I'm also realizing this is a carry-over from the older code.

Reply:
A `TODO` would be helpful.

Reply:
I'll added since it can't hurt.